Silicon/CommonSocPkg/Library/GpioV2Lib/PeiDxeSmmP2SbSidebandAccessLib.c

```diff
@@ -0,0 +1,936 @@
+/** @file
+  P2SB sideband access lib
+
+  Copyright (c) 2020 - 2023, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
```

Comment:
P2sbSideBandAccessLib should be a separate library if we need create this library, not part of Gpiov2Lib.
For some projects, they created PchP2sbLib which only provides the basic functions needed by SBL.
It looks Gpiov2Lib mainly use same one function.  It would have to link all the other functions to have a big size if using the P2SB_SIDEBAND_REGISTER_ACCESS ops. It could save size by just using normal library.